packages/zk-transfer-web/src/App.tsx

```diff
@@ -6,19 +6,25 @@ import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 
 import Homepage from './views/Home';
 import MakeTransaction from './views/MakeTransaction';
+import Democracy from './components/context/Democracy';
```

Comment:
Cool use of React contexts, I haven't learned about them before now. It looks like we can use this to render the page first, initialize the wallet / client, then update the page afterwards with the new/restored Ethereum address.

https://reactjs.org/docs/context.html#dynamic-context

What do you think?